grid_map_ros/package.xml

```diff
@@ -1,25 +1,38 @@
 <?xml version="1.0"?>
-<package format="2">
+<package format="3">
   <name>grid_map_ros</name>
-  <version>1.6.2</version>
+  <version>2.0.0</version>
   <description>ROS interface for the grid map library to manage two-dimensional grid maps with multiple data layers.</description>
   <maintainer email="mwulf@anybotics.com">Maximilian Wulf</maintainer>
   <maintainer email="ynava@anybotics.com">Yoshua Nava</maintainer>
   <license>BSD</license>
   <url type="website">http://github.com/anybotics/grid_map</url>
   <url type="bugtracker">http://github.com/anybotics/grid_map/issues</url>
   <author email="pfankhauser@anybotics.com">Péter Fankhauser</author>
-  <buildtool_depend>catkin</buildtool_depend>
-  <depend>roscpp</depend>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <build_depend>grid_map_cmake_helpers</build_depend>
+
+  <depend>rclcpp</depend>
```

Comment:
Could you also order here alphabetically?

Reply:
Wow, even by my standards that's detail oriented :-)

Reply:
Done!